DigitalOffice/Shared/Kernel/Broker/IOperationResult.cs

```diff
@@ -2,12 +2,18 @@
 
 namespace LT.DigitalOffice.Kernel.Broker
 {
+    /// <summary>
+    /// Represents interface which is a wrapper for broker response.
+    /// </summary>
     public interface IOperationResult<out T>
```

Comment:
If this is "unofficial" IOperationResult, delete Kernel at all and change with the Kernel from develop.

Reply:
Fixed